src/lib/path-utils.ts

```diff
@@ -0,0 +1,22 @@
+import * as path from 'path'
+
+export function relativeImportPathFromFile(from: string, to: string) {
+  // TODO is this OK on Windows?
```

Comment:
I need to test this. Not sure how the `path` module behaves in VSCode on Windows. It might use backslashes, which VSCode doesn't.

Reply:
It seems file paths generated by `path` (across this PR) will be used in two places:
1. `import` statement, which seems accepts Windows-style backslash path (from a quick look at issues like [this](https://github.com/babel/babel/issues/9765)), so I guess Posix/Windows path should both be supported.
2. As file path passed to [vscode.workspace.openTextDocument](https://code.visualstudio.com/api/references/vscode-api#Uri:~:text=A%20short%2Dhand%20for%20openTextDocument(Uri.file(fileName)).) which likely will do some path normalization.

So to conclude, the safer way should be use platform specific path (i.e. using `path` module to do everything about path, join(), relative() etc). I'm not sure about this, maybe the best way is to test it on a Windows machine. :)

Reply:
I have a Windows machine, I just usually only use it for gaming :-)

I'll install vscode on it and check the behavior.

Reply:
ChatGPT told me mixed path (with \ and / ) is supported in `import` statement and [vscode.workspace.openTextDocument](https://code.visualstudio.com/api/references/vscode-api#Uri:~:text=A%20short%2Dhand%20for%20openTextDocument(Uri.file(fileName)).), I decide to trust it this time and mark this as resolved...